app/models/new_york_time.rb

```diff
@@ -1,7 +1,8 @@
 class NewYorkTime
+  BUSINESS_NEWS_ENDPOINT='https://api.nytimes.com/svc/topstories/v2/business.json?api-key='
 
   def self.get_business_news
-    response = HTTParty.get("https://api.nytimes.com/svc/topstories/v2/business.json?api-key=#{ENV['NY_KEY']}")
+    response = HTTParty.get("#{BUSINESS_NEWS_ENDPOINT}#{ENV['NY_KEY']}")
     data_to_json = JSON.parse(response.body, symbolize_names: true)[:results]
     # sort news array [latest ... oldest]
     data_to_json.sort { |data1, data2| Time.parse(data2[:created_date]).to_i <=> Time.parse(data1[:created_date]).to_i }
```

Comment:
Instead of using command to explain this logic, we can extract it into a private method and name it as `sort_news`.